### PR TITLE
Class refactor and deprecation warning

### PR DESCRIPTION
**Pull request by mozky**
- Refactors class `PulicOrder` to `PublicOrder`
- Changes `.getStatus()` for `status()` as it was marked as deprecated by Netty.

### CODE REVIEW

File: src/main/java/com/bitso/BitsoOrderBook.java [hunk 1]
```diff
@@ -12,8 +12,8 @@
 public class BitsoOrderBook {
     private Date mOrderDate;
     private int mSequence;
-    private PulicOrder[] mAsks;
-    private PulicOrder[] mBids;
+    private PublicOrder[] mAsks;
+    private PublicOrder[] mBids;
```
Comment (VictorCruzIsc) [inline src/main/java/com/bitso/BitsoOrderBook.java]: I can't see any changes in this file. Could you please remove this file of the commit?
Best regards.

File: src/main/java/com/bitso/examples/BitsoWebSocketExample.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
 import com.bitso.Bitso;
 import com.bitso.BitsoOrder;
 import com.bitso.BitsoOrderBook;
-import com.bitso.BitsoOrderBook.PulicOrder;
+import com.bitso.BitsoOrderBook.PublicOrder;
```
Comment (VictorCruzIsc) [inline src/main/java/com/bitso/examples/BitsoWebSocketExample.java]: Also in this file, there are no visible changes.

File: src/main/java/com/bitso/websockets/BitsoWebSocket.java [hunk 3]
```diff
@@ -154,7 +154,7 @@ public void channelRead(ChannelHandlerContext ctx, Object msg)
 
             if (msg instanceof FullHttpResponse) {
                 FullHttpResponse response = (FullHttpResponse) msg;
-                throw new Exception("Unexpected FullHttpResponse (getStatus=" + response.getStatus() + ", content="
+                throw new Exception("Unexpected FullHttpResponse (getStatus=" + response.status() + ", content="
```
Comment (VictorCruzIsc) [inline src/main/java/com/bitso/websockets/BitsoWebSocket.java]: Nice catch, please update what I've asked in comments to proceed with your PR merge.

**Comment by AlexITC** [comment]
A misspelling was fixed, `PulicOrder` to `PublicOrder`.

**Comment by mozky** [comment]
Thats right, just that class name misspelling fix and the deprecation warning was removed.

**Comment by nvcruzhe** [comment]
@mozky Congrats, your pull request have been merged. Please feel free to continue working on this library or any in Bitso's repo.

**Pull request merged**